src/platform/packages/shared/kbn-es/src/utils/docker.ts

```diff
@@ -882,30 +890,40 @@ function getESClient(clientOptions: ClientOptions): Client {
  * Runs an ES Serverless Cluster through Docker
  */
 export async function runServerlessCluster(log: ToolingLog, options: ServerlessOptions) {
-  const image = getServerlessImage({
-    image: options.image,
-    tag: options.tag,
-  });
-  await setupDocker({ log, image, options });
+  await setupDocker({ log, options });
+
+  const image = getServerlessImage({ image: options.image, tag: options.tag });
+  await setupDockerImage({ log, image });
+
+  const uimaNodeNames = [];
```

Comment:
nit: there's a typo in this variable - `uiamNodes`

Reply:
Ah, yeah, thanks